docs/gambit/mutation-verifier.md

```diff
@@ -84,20 +108,21 @@ Soon, Certora will enable automatic notifications for asynchronous mutation test
 | Option                         | Description                                                                                                           |
 |:-------------------------------|:----------------------------------------------------------------------------------------------------------------------|
 | `--prover_conf`                | specify the Prover configuration file for verifying mutants                                                           |
-| `--gambit_conf`                | specify the configuration file for mutant generation                                                                  |
+| `--mutation_conf`              | specify the configuration file for mutant generation                                                                  |
 | `--num_mutants`                | request the mutant generator to generate a specific number of mutants. Defaults to 5                                  |
 | `--prover_version`             | specify the version of `certoraRun` to use for verification. Defaults to the installed version of `certoraRun`        |
-| `--server`                     | specify the server environment to run on. Defaults to the value specified in the file of `--prover_conf`, if exists   |
+| `--server`                     | specify the server environment to run on. Defaults to the value specified in the file of `--prover_conf`, if the field exists otherwise whatever `certoraRun` uses by default   |
```

Comment:
```suggestion
| `--server`                     | specify the server environment to run on. Defaults to the value specified in the file of `--prover_conf`, or the `certoraRun` default otherwise.   |
```